dialect_redshift.go

```diff
@@ -0,0 +1,19 @@
+package gorp
+
+type RedshiftDialect struct {
```

Comment:
so something we chatted about during story writing is that it's easy to confuse redshift's insert with say postgres' insert, and that we may find ourselves wanting to use the ID field that hasn't actually been set.  based on your work so far, is that unfounded?  or should we introduce something like `InsertRedshiftRow` to be explicit?

Reply:
Based on the tickets AC I created a `InsertRedshiftRow` on the `RSDS` struct and made the standard gorp `Insert` panic. Does that cover what you're asking here?

Reply:
yup, that does.  thanks!